.env-webserver-garbage-collector

```diff
@@ -0,0 +1,37 @@
+#
+# Explicit plugins DISABLED in the webserver to create a garbage-collector
+# Docs plugins config of services/web/server/src/simcore_service_webserver/application_settings.py
+#
+
+
+WEBSERVER_ACTIVITY=null
+WEBSERVER_CATALOG=null
+WEBSERVER_COMPUTATION=null
+WEBSERVER_DIAGNOSTICS=null
+#WEBSERVER_DIRECTOR_V2 from .env
+WEBSERVER_DIRECTOR=null
+WEBSERVER_EMAIL=null
+WEBSERVER_EXPORTER=null
+WEBSERVER_FRONTEND=null
+#WEBSERVER_GARBAGE_COLLECTOR explicit in

```

Comment:
This env-file configures a webserver instance to only act as a garbage collector, correct?
It is hard to infer that from this variable being commented out  (probably "true" is the default?, what does "explicit in" mean?) so maybe set it explicitly to true. Then every reader knows what is going on...

Reply:
ok, let me clarify all this for you offline